satellite/orders/endpoint.go

```diff
@@ -152,17 +149,21 @@ func (endpoint *Endpoint) Settlement(stream pb.Orders_SettlementServer) (err err
 				// who asked for this order: uplink (get/put/del) or satellite (get_repair/put_repair/audit)
 				if endpoint.satelliteSignee.ID() == *orderLimit.DeprecatedUplinkId {
 					uplinkSignee = endpoint.satelliteSignee
-				} else {
-					uplinkPubKey, err := endpoint.certdb.GetPublicKey(ctx, *orderLimit.DeprecatedUplinkId)
-					if err != nil {
-						log.Warn("unable to find uplink public key", zap.Error(err))
-						return status.Errorf(codes.Internal, "unable to find uplink public key")
-					}
-					uplinkSignee = &signing.PublicKey{
-						Self: *orderLimit.DeprecatedUplinkId,
-						Key:  uplinkPubKey,
-					}
 				}
+				// @TODO: KISHORE this portion of verifying uplink's public key is disabled until certdb is refactored
```

Comment:
We can completely remove the old logic, this shouldn't be needed anymore. Order limits using this logic should already have been sent to the satellite.

Reply:
updated. Removed the logic and added a comment to address the implementation in next PR.